server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -268,9 +268,12 @@ public void verifyReposThenStopServices() {
             if (blobStoreContext != null) {
                 blobStoreContext.forceConsistent();
             }
-            BlobStoreTestUtil.assertConsistency(
-                (BlobStoreRepository) testClusterNodes.randomMasterNodeSafe().repositoriesService.repository("repo"),
-                Runnable::run);
+            final PlainActionFuture<AssertionError> future = BlobStoreTestUtil.assertConsistencyAsync(
```

Comment:
I had to adjust the consistency check method to be more async now that the get snapshot info API is anync so I took the opportunity to simplify it's signature as well and remove the passing of the executor that was only needed for this spot before.